recipes/vtu11/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd, valid_min_cppstd
+from conan.tools.files import export_conandata_patches, get, copy, rm
```

Comment:
There's no patches and less if more :)

```suggestion
from conan.tools.files import get, copy, rm
```